### PR TITLE
Add dead links validation, fix found dead links

### PR DESCRIPTION
**Pull request by kandl**
JIRA: FET-949

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/924/ : SUCCESS in 1m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/613/ : SUCCESS in 7s

File: docs/10_vis__donut_chart.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ const style = { height: 300 };
 | filters | false | [IFilter[]](30_tips__filter_visual_components.md) | An array of filter definitions |
 | sortBy | false | [ISortItem[]](50_custom__result.md#sorting) | An array of sort definitions |
 | config | false | [IChartConfig](15_props__chart_config.md) | The chart configuration object |
-| locale | false | string | The localization of the chart. Defaults to `en-US`. For other languages, see the [full list of available localizations](https://github.com/gooddata/gooddata-sdk-ui-charts/tree/master/src/translations). |
+| locale | false | string | The localization of the chart. Defaults to `en-US`. For other languages, see the [full list of available localizations](https://github.com/gooddata/gooddata-ui-sdk/tree/release/libs/sdk-ui/src/base/localization/bundles). |
```
Comment (no23reason) [inline docs/10_vis__donut_chart.md:53]: Should probably point to master, the release branch changes in a non-intuitive ways for an external developer.
Same for other mentions of `locale` in other chart types.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/925/ : SUCCESS in 1m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/614/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/926/ : SUCCESS in 1m 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/615/ : SUCCESS in 7s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR475

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/928/ : SUCCESS in 1m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/179/ : SUCCESS in 1m 48s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/74/ : SUCCESS in 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/269/ : SUCCESS in 18s

**Pull request merged**